wdn/templates_4.0/scripts/search.js

```diff
@@ -23,117 +23,145 @@ define(['jquery', 'wdn', 'require', 'modernizr', 'navigation'], function($, WDN,
 					submitted = false,
 					postReady = false,
 					autoSubmitTimeout,
-					searchOrigin = '//www1.unl.edu',
-					siteHomepage = nav.getSiteHomepage();
-
-				/**
-			     * Add the experimental text-to-speech
-			     */
-	            domQ.attr('x-webkit-speech', 'x-webkit-speech');
-
-				var localSearch = getLocalSearch();
-				if (localSearch) {
-					// Change form action to the local search
-					var qsPos = localSearch.indexOf('?'), hashes, hash, htmlUpdate = $();
-					if (qsPos > -1) {
-						hashes = localSearch.slice(qsPos + 1).split('&');
-						for (var i = 0; i < hashes.length; i++) {
-							hash = hashes[i].split('=');
-							htmlUpdate = htmlUpdate.add($('<input>', {
-								type: "hidden",
-								name: hash[0],
-								value: decodeURIComponent(hash[1])
-							}));
+					searchHost = 'www1.unl.edu', // domain of UNL Search app
+					searchPath = '/search/', // path to UNL Search app
+					searchOrigin = window.location.protocol + '//' + searchHost,
+					searchAction = searchOrigin + searchPath,
+					allowSearchParams = ['u', 'cx'],  // QS Params allowed by UNL Search app
```

Comment:
Too allow developers to manually provide a "search root" for their site/sub-site in cases where the default logic (using the site homepage) would result in unexpected/poor local search performance. For example, when the home page of a site is at a file URL rather than a directory.

This is just a stricter enforcement of the previously documented was of defining your local search: http://www1.unl.edu/wdn/wiki/Local_Site_Links#search